filters/filter_24_QuickFixes/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "filterId": 24,
+  "name": "AdGuard Quick Fixes filter",
+  "description": "Filter to quickly resolve content filtering issues on popular websites without updating the extension.",
+  "timeAdded": 1724747105555,
+  "homepage": "https://adguard.com/kb/general/ad-filtering/adguard-filters/",
+  "expires": "5 days",
+  "displayNumber": 1,
+  "groupId": 1,
+  "subscriptionUrl": "https://filters.adtidy.org/extension/chromium/filters/24.txt",
```

Comment:
the filter list will not be built for `chromium` due to `platformsIncluded` below

Reply:
Yep, that's right. Exchanged to this `https://filters.adtidy.org/extension/chromium-mv3/filters/24.txt`, thank you.